src/py/mat3ra/made/tools/build/defect/builders.py

```diff
@@ -44,6 +44,7 @@
     PointDefectPairConfiguration,
 )
 from .factories import DefectBuilderFactory
+from pymatgen.analysis.adsorption import AdsorbateSiteFinder
```

Comment:
third_party

Reply:
we decided to move those sites functionalities into other task